src/pages/ReimbursementAccount/BusinessInfo/substeps/AddressBusiness.tsx

```diff
@@ -35,7 +25,7 @@ const INPUT_KEYS = {
 
 const REQUIRED_FIELDS = [companyBusinessInfoKey.STREET, companyBusinessInfoKey.CITY, companyBusinessInfoKey.STATE, companyBusinessInfoKey.ZIP_CODE];
 
-const validate = (values) => {
+const validate = (values: OnyxCommon.Errors) => {
```

Comment:
Are you sure that it's right to say that values passed to the validate function have `OnyxCommon.Errors` type? I think you create a separate type for it. Also please ass return type

Reply:
`getFieldRequiredErrors` has the `values` typed as `OnyxCommon.Errors`, hence the typing here but it's more appropriate to be the return of this fn